apps/wakunode2/app.nim

```diff
@@ -596,6 +619,8 @@ proc startRestServer(app: App, address: ValidIpAddress, port: Port, conf: WakuNo
       app.node.subscribe((kind: ContentSub, topic: contentTopic), some(autoHandler))
 
     installRelayApiHandlers(server.router, app.node, cache)
+  else:
+    notInstalledTab["relay"] = "/relay endpoints are not available"
```

Comment:
Do we have protocol names anywhere as constants? It would be better/nicer than having plain strings as keys (although also probably ok for this singular use case)

Reply:
You're right with that, but currently there are no such consts only for each endpoint paths.
The other option But still would not be more clear coded as we don't have mechanism to structure request handling in hierarchy, so it would be error prone even to duplicate similar code to each handler.
For now I would keep it short and simple here but will add some better response message here I see I missed it though.